src/keysBuilder/extractTSMarkerKeys.ts

```diff
@@ -0,0 +1,20 @@
+import { tsquery } from '@phenomnomnominal/tsquery';
+import * as ts from 'typescript';
+import { buildKeysFromASTNodes } from './buildKeysFromASTNodes';
+
+export function extractMarkerKeys(ast): { key: string; lang: string }[] {
+  const [importNode] =
+    tsquery(ast, `ImportDeclaration:has([text=@ngneat/transloco-keys-manager/marker])`) ||
```

Comment:
Use `ImportDeclaration:has([text=/^@ngneat\x2Ftransloco-keys-manager/])`.
Also, add a this is link: https://github.com/estools/esquery/issues/68 as a comment above this line.

Reply:
done